Sources/protoc-gen-swift/ExtensionGenerator.swift

```diff
@@ -119,7 +121,7 @@ struct ExtensionGenerator {
         if comments != "" {
             p.print(comments)
         }
-        p.print("public var \(swiftFieldName): \(apiType) {\n")
+        p.print("\(generatorOptions.visibilitySourceSnippet)var \(swiftFieldName): \(apiType) {\n")
```

Comment:
@tbkka the extensions where the ones I'm the least sure about, are these correct?